### PR TITLE
Adds sponsors and popular projects to the homepage

### PR DESCRIPTION
**Pull request by Jurj-Bogdan**
|    Q          |   A
|-------------- | ------
| New Feature   | yes
| RFC           | yes

### Description

An initial step in implementing the new homepage design, as discussed in #162. Added the "sponsor" and "popular projects" sections suggested in the comments, albeit on a white background for logo visibility.

Open to any suggestions/changes going forward with the issue at hand

### CODE REVIEW

**Comment by arhimede** [comment]
Preview here:
https://preview-lf246gy-2ja7ciew2nbkm.us-2.platformsh.site/

File: config/autoload/home-page-content.global.php [hunk 1]
```diff
@@ -0,0 +1,123 @@
...
+            'name' => 'Magento',
+            'logo' => '/images/sponsors/magento.png',
+        ],
+        [
+            'name' => 'Owncloud',
+            'logo' => '/images/sponsors/owncloud.png',
+        ],
+        [
+            'name' => 'Drupal',
+            'logo' => '/images/sponsors/drupal.png',
```
Comment (froschdesign) [inline config/autoload/home-page-content.global.php:39]: The Drupal logo should also contain the wordmark: https://www.drupal.org/about/media-kit/logos

File: config/autoload/home-page-content.global.php [hunk 2]
```diff
@@ -0,0 +1,123 @@
...
+            'name' => 'Apidemia',
+            'logo' => '/images/vendors/apidemia.png',
+            'url'  => 'https://www.apidemia.com',
+            'text' => 'Apidemia is a web development company, offering software development and consulting. Since 2005, we have focused on the PHP ecosystem, and our tools of trade have been Zend Framework 1, 2, and 3, and then Laminas and Mezzio. We created DotKernel, a collection of PSR-7 Middleware applications built on top of the Mezzio microframework and Laminas components.',
+        ],
+    ],
+    'projects-using-components' => [
+        [
+            'name' => 'Magento',
+            'logo' => '/images/sponsors/magento.png',
```
[the remark this answers is not in the document]
Comment (froschdesign) [reply config/autoload/home-page-content.global.php:31]: The Magento logo should also contain the name for better identification.

**Review by froschdesign — CHANGES_REQUESTED** [review]
A good first step.

**Comment by arhimede** [comment]
preview updated 
https://preview-lf246gy-2ja7ciew2nbkm.us-2.platformsh.site/

**Comment by froschdesign** [comment]
@weierophinney 
Do you see any legal problems? I remember that this was once a topic.

**Comment by weierophinney** [comment]
@froschdesign Nothing leaps out as problematic to me. :ship:

**Review by froschdesign — APPROVED** [review]

**Pull request merged**